{{cookiecutter.__src_folder_name}}/.github/workflows/test_postgres.yml

```diff
@@ -37,8 +46,8 @@ jobs:
         with:
           {% raw %}
           python-version: ${{ matrix.python_version }}
-          architecture: x64
           {% endraw %}
+          architecture: x64
```

Comment:
Is this going to be a problem when testing against ARM?

Reply:
This change is not needed but I asked myself why are we disabling interpretation for `architecture: x64`
it's not related to jinja syntax so, it's safe to do it like that and the wrapping was not needed.